packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -557,7 +559,7 @@ export class GarbageCollector implements IGarbageCollector {
 
         // Inactive timeout must be greater than sweep timeout since a node goes from active -> inactive -> sweep ready.
         if (this.sweepTimeoutMs !== undefined && this.inactiveTimeoutMs > this.sweepTimeoutMs) {
-            throw new UsageError("inactive timeout should not be greated than the sweep timeout");
+            throw new UsageError("inactive timeout should not be greater than the sweep timeout");
```

Comment:
What's the difference between UsageError and an Assert?

Reply:
assert means we screwed up (so I use that in our internal class that only we call).  UsageError means the caller screwed up (so we put that here where anyone using GarbageCollection will face it if they give bad values)